src/coreclr/tools/aot/crossgen2/crossgen2.csproj

```diff
@@ -10,7 +10,8 @@
     <!-- Trimming is not currently working, but set the appropriate feature flags for NativeAOT -->
     <PublishTrimmed Condition="'$(NativeAotSupported)' == 'true'">true</PublishTrimmed>
     <RuntimeIdentifiers Condition="'$(RunningPublish)' != 'true' and '$(DotNetBuildFromSource)' != 'true'">linux-x64;linux-musl-x64;linux-arm;linux-musl-arm;linux-arm64;linux-musl-arm64;freebsd-x64;osx-x64;osx-arm64;win-x64;win-x86;win-arm64;win-arm</RuntimeIdentifiers>
-    <RuntimeIdentifiers Condition="'$(DotNetBuildFromSource)' == 'true'">$(PackageRID)</RuntimeIdentifiers>
+    <RuntimeIdentifier Condition="'$(DotNetBuildFromSource)' == 'true'">$(PackageRID)</RuntimeIdentifier>
```

Comment:
This change shouldn't be necessary because it should be fixed in the sdk - https://github.com/dotnet/sdk/issues/27985